repo/packages/K/kafka/0/config.json

```diff
@@ -1,6 +1,35 @@
 {
-  "type": "object",
-  "properties": {},
-  "additionalProperties": false,
-  "required": []
+    "type": "object",
+    "properties": {
```

Comment:
Yes. You can use "kafka/". We are also in the process of improving the Marathon package so that we can use that as an example for other packages to follow. See: https://github.com/mesosphere/universe/pull/102